src/ui/pages/Notifications/NotificationItem.tsx

```diff
@@ -1,27 +1,26 @@
-import { IonItem, IonLabel, IonIcon } from "@ionic/react";
-import { ellipsisHorizontal } from "ionicons/icons";
-import i18next from "i18next";
-import { useEffect, useState } from "react";
-import KeriLogo from "../../assets/images/KeriGeneric.jpg";
-import { getConnectionsCache } from "../../../store/reducers/connectionsCache";
-import { useAppSelector } from "../../../store/hooks";
+import { IonIcon, IonItem, IonLabel } from "@ionic/react";
+import { t } from "i18next";
+import { ellipsisHorizontal, fingerPrintOutline } from "ionicons/icons";
+import { MouseEvent, useCallback, useEffect, useState } from "react";
+import { Trans } from "react-i18next";
+import { Agent } from "../../../core/agent/agent";
 import {
   KeriaNotification,
   NotificationRoute,
 } from "../../../core/agent/agent.types";
-import { timeDifference } from "../../utils/formatters";
-import { Agent } from "../../../core/agent/agent";
 import { MultiSigIcpRequestDetails } from "../../../core/agent/services/identifier.types";
+import { useAppSelector } from "../../../store/hooks";
+import { getConnectionsCache } from "../../../store/reducers/connectionsCache";
+import KeriLogo from "../../assets/images/KeriGeneric.jpg";
+import MultisignReferIcon from "../../assets/images/multisign-icon.jpg";
```

Comment:
Let's not use `jpg` and just use the icons from https://ionic.io/ionicons.

Also, these icons should be named after the categories in the filters above (Identifiers - Credentials) rather than the specific use case (mutli-sig request can be just one use for it).